src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -292,6 +293,7 @@ public long getMCreatorVersion() {
 
 		generator.close();
 		fileManager.close();
+		ExternalTexture.invalidateCache(this);

```

Comment:
Thinking again, this won't work correctly either, as new workspace can be open while another is open. This cache will need to be per-workspace and be soft-cached, otherwise this will not work again